puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/build/build_rpm_rubygem-katello_nightly.yaml

```diff
@@ -0,0 +1,32 @@
+- job:
+    name: build_rpm_rubygem-katello_nightly
+    node: fast
+    properties:
+      - tfm-build-discarder
+    triggers:
+      - timed: '0 18 * * *'
```

Comment:
What about keeping https://github.com/theforeman/foreman-infra/pull/685/files#diff-0141dbd04e4380140b3d3ea37281a53eL24 so that this builds a new RPM every time a change is introduced (PR is merged) ?

Reply:
yeah, we can do that

Reply:
updated